proposals/NNNN-vehicle-type-filter.md

```diff
@@ -0,0 +1,435 @@
+
+### App name shown on notifications in Android
+
+The exclusive apps should not host `SDL Router service` on unsupported SDL systems. The android system requires all apps using foreground services to show a notification. The SDL enabled android app can have two foreground services. The `SDL router service` is started by the android proxy and another foreground service will be started by an SDL enabled app.
+
+The `SDL router service` has no information on what head unit it is connecting with when a transport connection is made. Other SDL apps bind to available router service and use this connection. This behavior could lead to cases where a proprietary app creates an `SDL router service` when connected to an unsupported SDL enabled system. This started service will need to show a foreground notification with the app name, giving the user the impression that the proprietary app is working with an unsupported SDL enabled system.
+
+For example, A proprietary app of an OEM creates Router service and the mobile device is connected to another OEMs' SDL enabled IVI system. The user would perceive that the proprietary app is doing something in the background, when connected to SDL enabled IVI system from another OEM.
+
+Please refer below screenshots.
+
+- Notification 1: Created by OEM App.
+- Notification 2: Created by SDL Android SDK for `SDL Router Service`.
+
+__Note the app name being shown in both notifications. In the below picture, the `SDL App` is using the “SDL Router Service” created by the `OEM App`.__
+
+<img src="../assets/proposals/NNNN-vehicle-type-filter/two_noti.png" alt="App name in notifications" width="500"/>
+
+
+The only way to remove notifications from OEM App when it connects to another vehicle is to force stop both the services so, that both notifications are removed. In this case, other apps connected through the `OEM Apps'` router service will also get disconnected. This is not a recommended approach. If the OEM app wants to unregisters itself from the SDL enabled IVI system, it will stop the service created by an app. The `SDL router service` from the OEM app will keep running.
```

Comment:
```suggestion
The only way to remove notifications from an OEM App when it connects to another vehicle is to force stop both the services so that both notifications are removed. In this case, other apps connected through the OEM App's router service will also get disconnected. This is not a recommended approach. If the OEM app wants to unregisters itself from the SDL enabled IVI system, it will stop the service created by an app. The `SDL router service` from the OEM App will keep running.
```